docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,110 @@
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="https://www.researchgate.net/profile/Chaiwat-Sakul/publication/228949121/figure/fig2/AS:300654661783574@1448693065750/The-previous-square-root-circuit-1.png" width="500">
+  <!-- Default Legend (popover only)-->
+  <a-point x="25%" y="50%" body="There is only text here" label="1"/>
+  <!-- Set Legend to bottom only -->
+  <a-point x="50%" y="50%" body="Hovering over this does nothing" label="2" legend="bottom"/>
+  <!-- Set Legend to both -->
+  <a-point x="75%" y="50%" body="There is text here and when you hover over it" label="3" legend="both"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+\***\*`<a-point>` Options\*\***
+
+| Name      | Type     | Default   | Description                                                                                                               |
+| --------- | -------- | --------- | ------------------------------------------------------------------------------------------------------------------------- |
+| x         | `String` | `''`      | How to position the x-coordinate of the point.<br>Supports range of values from `0%` to `100%`.                           |
+| y         | `String` | `''`      | How to position the y-coordinate of the point.<br>Supports range of values from `0%` to `100%`.                           |
+| body      | `String` | `''`      | The content to be inserted into the body of the text.<br>The body will be omitted if this is not provided                 |
+| header    | `String` | `''`      | The content to be inserted into the header of the text.<br>The header will be omitted if this is not provided             |
+| trigger   | `String` | `click`   | How the Popover is triggered.<br>Supports: `click`, `focus`, `hover`, or any space-separated combination of these.        |
+| placement | `String` | `top`     | How to position the Popover.<br>Supports: `top`, `left`, `right`, `bottom`.                                               |
+| label     | `String` | `''`      | The content to be shown over the point.<br>The label will be omitted if this is not provided                              |
+| size      | `String` | `'40'`    | The size of the point in pixels.                                                                                          |
+| color     | `String` | `'green'` | The color of the point.<br>Supports any color in the CSS color format. E.g. `red`, `#ffffff`, `rgb(66, 135, 245)`, etc... |
+| opacity   | `String` | `'0.3'`   | The opacity of the point.<br>Supports range of values from `0` to `1`.                                                    |
+| legend     | `String` | `'popover'`      | The presence of a legend to be displayed.<br>Supports: `popover`, `bottom`, `both`.                          |
```

Comment:
> * `content` instead of `header` to match the rest of the pop-ups?

Did you mean `content` instead of `body`?

> * The name `legend` feels a little confusing to me (if it's at the bottom, I would call it a legend, but if it's a popover, I would think of it more of an annotation/label/pop-up). Though I'm not sure what would be a better alternative 🤔

Yeah, I also think that there could be a better name. But `legend` is indicative of what it actually is right haha.

> * Regarding the options for `legend`: `bottom` is one of the possible values of `placement`, so we may want to avoid having that as one of the options for `legend`. Maybe `caption` or `footnote` could be used instead of `bottom`? (Or any other suggestions)

Ok, maybe we can go with footnote.

> * We can add an `alt` attribute too for accessibility purposes (to set the `alt` attribute of the `<img>` tag)! Would also allow an alternative text to be shown if the image fails to load for some reason.

alt is actually already supported, will have to add the options for the `<annotate>` class

Reply:
> Yeah, I also think that there could be a better name. But `legend` is indicative of what it actually is right haha.

Ah ok, I just looked up "legend" and realised there are multiple meanings to the term. 
To me `bottom` is a legend, `popover` is a popover/pop-up, and collectively not sure what's the best term to call them.